modules/aws_mysql/tf_module/variables.tf

```diff
@@ -37,3 +37,8 @@ variable "multi_az" {
   type    = bool
   default = false
 }
+
+variable "identifier" {
+  type    = string
+  default = null
```

Comment:
let's not add `default = null` since we would always set it, similar to env_name

Reply:
I tried setting this to a default value which we use, but the interpolation is not allowed for variables in terraform and we do need this as `null` in order to check what name to set.